src/current/_data/releases.yml

```diff
@@ -9042,7 +9042,7 @@
   release_date: '2025-05-05'
```

Comment:
Need to update the release date.

Reply:
Do we actually want to update the release date to be current? Or retroactive for that matter? The original 25.1.6 cloud release date was Apr 30th, and I figured that the updated SHA would be invisible for Cloud customers.

Reply:
Ah so, did v25.1.6 actually release on cloud on 4/30? If that's the case, I would change to that.